components/apimgt/org.wso2.carbon.apimgt.keymgt/src/main/java/org/wso2/carbon/apimgt/keymgt/handlers/DefaultKeyValidationHandler.java

```diff
@@ -267,7 +270,11 @@ private AccessTokenInfo getAccessTokenInfo(TokenValidationContext validationCont
                 CacheProvider.getGatewayIntrospectCache().put(validationContext.getAccessToken(), tokenInfo);
                 return tokenInfo;
             } else {
+                AccessTokenInfo tokenInfo = new AccessTokenInfo();
+                tokenInfo.setTokenValid(false);
+                tokenInfo.setErrorcode(900901);
```

Comment:
Shall we keep the the error code in a constant  file and reuse? @binodmx

Reply:
This is done by https://github.com/wso2/carbon-apimgt/pull/10968